config/runtime/kbuild.jinja2

```diff
@@ -19,29 +21,131 @@ KBUILD_PARAMS = {
     'arch': '{{ arch }}',
     'compiler': '{{ compiler }}',
     'defconfig': '{{ defconfig }}',
-    'fragments': [],
+{%- if fragments %}
+    'fragments': {{ fragments }}
+{%- else %}
+    'fragments': []
+{%- endif %}
 }
 {%- endblock %}
 
 {% block python_job -%}
+
```

Comment:
Oh I only see this now, I guess I understand a bit better why you didn't like Jinja2.  Yes the idea is really not to have the build logic in Jinja2 but in the `kernelci` Python package instead.  I guess this is a good first stepping stone anyway to prepare the code and then merge it somehow in `kernelci.build` or elsewhere.